### ISSUE TITLE
PHP: Eclim does not put full namespace's path

### ISSUE THREAD

**Issue by paprykarz** [ervandew/eclim#441]
Hi Eric

Please see the constructor of Image class in https://github.com/paprykarz/eclimtest and try to complete its argument. In this situation I get menu with namespaces:

     Image     Image - Core\Domain\Models
     ImageSize ImageSize - Core\Domain\ValueObjects 

Selecting `ImageSize` works not completely: class name is completed but its path loses - it does not appear both directly and in "use" statement. 

Regards

**Comment by kminh** [comment]
Could be a duplicate of https://github.com/ervandew/eclim/issues/414

**Comment by przepompownia** [comment]
Yes, if we would also say about appending "use" clause below namespace declaration, but not only about inserting full path at the place of completion. See also par. 3 in  https://www.jetbrains.com/phpstorm/help/creating-imports.html#completeShortClassName
